### DISCUSSION TITLE
Sd card support for dsiware

### DISCUSSION THREAD

**Discussion by ghost** [rafaelvcaetano/melonDS-android#759] [Ideas]
Add dsi sd card support and unlaunch nand support

**Comment by Damaj301damaj-lol** [comment]
Why would you like unlaunch on an emulator?

**Reply by ghost** [reply]
I would like sd card support to expand the number of games to install
And unlaunch to play out ot region games and theme

**Reply by Kmog2185** [reply]
This would be a great addition to this emulator.
